### PR TITLE
Feature/profile page

### PR DESCRIPTION
**Pull request by Tommybui28**
Can't see if it works because API Key changed :((((

### CODE REVIEW

**Review by sahilpatel1906 — APPROVED** [review]

File: capstone_frontend/src/components/Profile.jsx [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+const Profile = (driver) => {
+
     return (
-        <p>Profile:</p>
+        <>
+            <h3>My Profile</h3>
+            <p>{driver.name}</p>
+            <p>{driver.emailAddress}</p>
+            <p>{driver.phone}</p>
+
```
Comment (Aebel-Shajan) [inline capstone_frontend/src/components/Profile.jsx:11]: Are we gonna display the vehicles?

File: capstone_backend/src/main/java/com/example/Capstone_Backend/components/DataLoader.java [hunk 2]
```diff
@@ -28,15 +28,15 @@ public class DataLoader implements ApplicationRunner {
     @Override
     public void run(ApplicationArguments args) throws Exception {
 
-        Driver jean = new Driver("Jean", "jean@test.com", "+44339284023");
+        Driver jean = new Driver("Jean", "jean@test.com", "+44339284023", "Trolley", "JE08 ANN");
         driverRepository.save(jean);
-        Driver aebel = new Driver("Aebel", "aebelshajan.work@gmail.com", "+447577563733");
+        Driver aebel = new Driver("Aebel", "aebelshajan.work@gmail.com", "+447577563733", "Clown Car", "AE16 BEL");
```
Comment (Aebel-Shajan) [inline capstone_backend/src/main/java/com/example/Capstone_Backend/components/DataLoader.java:33]: Excuse me !!??? this is ridiculous, i should be driving a ferrarii

**Review by Aebel-Shajan — CHANGES_REQUESTED** [review]
Add vehicles to profile page. Give me a ferrari pls

File: capstone_frontend/src/components/Profile.jsx [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+            
+            <h4>My Details</h4>
+            <p>{driver.name}</p>
+            <p>{driver.emailAddress}</p>
+            <p>{driver.phone}</p>
+
+            <h4>My Vehicle</h4>
+            <p>{driver.vehicleType}</p>
+            <p>{driver.licensePlateNumber}</p>
+
```
Comment (ney601) [inline capstone_frontend/src/components/Profile.jsx:15]: Thanks for adding the vehicle details

File: capstone_backend/src/main/java/com/example/Capstone_Backend/components/DataLoader.java [hunk 4]
```diff
@@ -28,15 +28,15 @@ public class DataLoader implements ApplicationRunner {
     @Override
     public void run(ApplicationArguments args) throws Exception {
 
-        Driver jean = new Driver("Jean", "jean@test.com", "+44339284023");
+        Driver jean = new Driver("Jean", "jean@test.com", "+44339284023", "Mercedes F1-Car", "JE08 ANN");
         driverRepository.save(jean);
-        Driver aebel = new Driver("Aebel", "aebelshajan.work@gmail.com", "+447577563733");
+        Driver aebel = new Driver("Aebel", "aebelshajan.work@gmail.com", "+447577563733", "Lightning McQueen Car", "AE16 BEL");
```
Comment (Aebel-Shajan) [inline capstone_backend/src/main/java/com/example/Capstone_Backend/components/DataLoader.java:33]: Good enough....

**Review by Aebel-Shajan — APPROVED** [review]

**Review by ney601 — APPROVED** [review]
Thanks for the G-Wagon!

**Review by Jean-005 — APPROVED** [review]
Nice work!

**Pull request merged**